src/app/components/audio-recordings/pages/download/download.component.html

```diff
@@ -287,7 +337,7 @@ <h3>6. Run the script</h3>
 
   <p>
     <code>
-      ./download_audio_files.ps1 -auth_token "INSERT_AUTH_TOKEN_HERE
+      ./download_audio_files.ps1 -auth_token "INSERT_AUTH_TOKEN_HERE"
```

Comment:
maybe we could simplify this step if provided a pre-formatted command... we might have to toggle visibility but it would remove all of step 5 and avoid typing in the terminal

Reply:
We would have to lock the page to logged in users only

Reply:
or fallback to what we have now?

or if not logged in show "login to see auth token"